server/tests/routes/auth.test.js

```diff
@@ -0,0 +1,93 @@
+import request from 'supertest-as-promised';
+import httpStatus from 'http-status';
+import app from '../../../index';
+import { fixtures, build } from '../../utils/factories';
+import User from '../../models/user';
+
+const authUrl = '/api/v1/auth';
+
+describe('## Auth API', () => {
+  let user;
+
+  beforeEach((done) => {
+    user = fixtures.user();
+    build('user', user)
```

Comment:
This and the line above can be combined into 1, and get rid of the unnecessary assignment.

`build('user', fixtures.user())`